tests/components/yeelight/__init__.py

```diff
@@ -94,6 +95,15 @@ def _mocked_bulb(cannot_connect=False):
     bulb.bulb_type = BulbType.Color
     bulb.last_properties = PROPERTIES
     bulb.music_mode = False
+    bulb.async_listen = MagicMock(return_value=Future())
+    bulb.async_listen.return_value.set_result(True)
```

Comment:
This could probably just be an AsyncMock

Reply:
done